test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules.lua

```diff
@@ -215,6 +215,17 @@ function c.getAnotherModuleControlData(pModuleType)
   return struct
 end
 
+
+function c.getSettableModuleControlData(pModuleType)
```

Comment:
@GetmanetsIrina I think we also have to use parent function ```getSettableModuleControlData()``` function from ```commonRC``` module since we run 'Set...' scripts not only for ```AUDIO```, but for the ```CLIMATE, AUDIO, LIGHT, HMI_SETTINGS``` modules